docs/rules/prefer-numeric-literals.md

```diff
@@ -0,0 +1,41 @@
+# disallow `parseInt` in favor of binary, octal, and hexadecimal literals (prefer-numeric-literals)
+
+Binary and octal literals were introduced in ES6. Prior to this, `parseInt` was used to turn binary and octal strings into integers. This rule encourages use of binary, octal, and hexadecimal literals instead of `parseInt`.
+
+```js
+0b111110111 === 503;
+0o767 === 503;
+```
+
+## Rule Details
+
+This rule disallows `parseInt` if it is called with a radix option of 2 (binary), 8 (octal), or 16 (hexadecimal).
```

Comment:
I'd love to see this be a bit more explicit - i.e. something along the lines of `This rule disallows `parseInt()` when called with a string and a radix option of 2 (binary), 8 (octal), or 16 (hexadecimal) in favor of the respective binary, octal, or hexadecimal numeric literal`.

It's a bit confusing as it stands now because there is a correct code option (`parseInt(foo, 2);`) that has a radix option of `2` below.